docs/concepts/typings.md

```diff
@@ -4,4 +4,103 @@ title: Type hints
 
 # Type hints
 
-...
+Type hints are a modern feature of Python available since 3.5 whose existence is heavily influenced by the features of type-safe languages such as Rust. To learn more about type hints in Python, see [Real Python’s Type hinting walkthrough](https://realpython.com/lessons/type-hinting/).
+
+When using Strawberry to build graphQL APIs, as was shown in [Schema basics](https://strawberry.rocks/docs/general/schema-basics), type hints are required within classes decorated by `@strawberry.type` & `@strawberry.input` and functions decorated by `@strawberry.field` & `strawberry.mutation`. These type hints are sourced as the keywords `str`, `int`, `float`, and from packages imported directly from the Python standard libraries `typing`, that has been available since Python 3.5, `datetime`, and `decimal`.
```

Comment:
> These type hints are sourced as the keywords `str`, `int`, `float`, and from packages imported directly from the Python standard libraries `typing`, that has been available since Python 3.5, `datetime`, and `decimal`.

The wording here is a bit confusing. I think what you're saying is that GraphQL types are directly created using the types you listed, but it sounds like these are the only allowed hints. We support more (esp. using types defined with `@strawberry.type`) so we should make this more clear.

Reply:
Good point, I will try to make that as clear as possible